pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -353,10 +353,10 @@ type reconstructedVolume struct {
 	volumeSpec          *volumepkg.Spec
 	outerVolumeSpecName string
 	pod                 *v1.Pod
-	attachablePlugin    volumepkg.AttachableVolumePlugin
 	volumeGidValue      string
 	devicePath          string
 	mounter             volumepkg.Mounter
+	deviceMounter       volumepkg.DeviceMounter
```

Comment:
I replace `AttachableVolumePlugin` with `DeviceMounter` (DeviceMountableVolumePlugin) because:

1) `AttachableVolumePlugin` wraps `DeviceMounter`, so all plugins implementing `AttachableVolumePlugin` implement `DeviceMounter` and can get device mount path via `GetDeviceMountPath()` method
2) local plugin (and only this plugin) does not implement `AttachableVolumePlugin` but implements `DeviceMountableVolumePlugin`